extensions/amp-analytics/0.1/events.js

```diff
@@ -171,6 +187,17 @@ function isVideoTriggerType(triggerType) {
   return triggerType.startsWith('video');
 }
 
+/**
+ * @param {string} triggerType
+ * @return {boolean}
+ */
+function isCustomBrowserTriggerType(triggerType) {
+  const values = Object.keys(BrowserEventType).map((key) => {
+    return BrowserEventType[key];
+  });
+  return values.indexOf(triggerType) > -1;
+}
```

Comment:
Like @dmanek mentioned, this is not the best way to do this. If you use `isEnumValue`, this will compile into the most efficient form.

```suggestion
import {isEnumValue} from '#core/types';

/**
 * @param {string} triggerType
 * @return {boolean}
 */
function isCustomBrowserTriggerType(triggerType) {
  return isEnumValue(BrowserEventType, triggerType);
}
```